app/common/util/dateUtil.js

```diff
@@ -65,6 +65,13 @@ Bahmni.Common.Util.DateUtil = {
     getDateWithoutTime: function(datetime){
         return datetime?moment(datetime).format("YYYY-MM-DD"):null;
     },
+    
+    getDateInMonthsAndYears : function (date, format){
+        var format = format || "MMM YY";
```

Comment:
@ningosi because javascript is always pass by value, it make sense to say that the variable was declared in the parameter, but there is a guideline that says [always declare local variables](http://www.w3schools.com/js/js_best_practices.asp), in this particular case we have a primitive/string variable being passed, but when a variable refers to an object (including arrays), the "value" is a reference to the object. So:
```
var a = {name: "global"}; 
 function hello (y) {
        y.name = "local";
}
hello(a);
//in this line a.name => "local"
```
I suggest that as a guideline we declare local variables using `var`